src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.tsx

```diff
@@ -123,6 +126,29 @@ export const TranscriptsListItemInfo = (
     objectId: props.gene.id
   });
 
+  const buildUrlForProteinView = (transcriptId: string) => {
```

Comment:
the function says "buildUrl", but what it's building is a link element.

Reply:
Renamed it to `getLinkToProteinView`